data/gaia_j0452-36b.json

```
@@ -15,14 +15,48 @@
     "CompanionRelationships": [
         {
             "companion_name": "Gaia EDR3 4818823636756117504",
-            "projected_separation_arcsec": null,
+            "projected_separation_arcsec": 115.3,
             "projected_separation_error": null,
             "relationship": "Child",
             "comments": "Spectral Type: esdM1",
             "reference": "Zhan19.1423",
             "other_companion_names": "Gaia J0452-36A"
         }
-    ],  
+    ], 
+    "ModeledParameters": [
+        {
+            "parameter": "T eff",
+            "value": 2600.0,
+            "value_error": 100.0,
+            "unit": "K",
+            "comments": null,
+            "reference": "Zhan19.1423"
+        },
+        {
+            "parameter": "log g",
+            "value": 5.5,
+            "value_error": 0.2,
+            "unit": "dex",
+            "comments": null,
+            "reference": "Zhan19.1423"
+        },
+        {
+            "parameter": "mass",
+            "value": 0.0855,
+            "value_error": 0.0014,
+            "unit": "M_jup",
+            "comments": null,
+            "reference": "Zhan19.1423"
+        },
+        {
+            "parameter": "metallicity",
+            "value": -1.4,
+            "value_error": 0.2,
+            "unit": "Fe/H",
```

unit should be "dex"
```suggestion
            "unit": "dex",
```